src/components/Button/Button.tsx

```diff
@@ -46,7 +46,7 @@ export class Button extends React.Component<IButtonProps, IButtonState> implemen
     });
 
     const iconSpan = icon && (buttonType === ButtonType.command || buttonType === ButtonType.hero || buttonType === ButtonType.icon)
-      ? <span className='ms-Button-icon'><i className={`ms-Icon ms-Icon--${icon}`}></i></span>
+      ? <span className='ms-Button-icon'><i className={'ms-Icon ms-Icon--${icon}'}></i></span>
```

Comment:
Reason I changed the backticks was because they were causing this to happen in the Sources tab in chrome

![image](https://cloud.githubusercontent.com/assets/1434956/19895908/c8082a36-a00f-11e6-8789-2a5a655e360b.png)

But you're right, it does break the code :-\

Oh well. I just wanted to be able to debug the code below it. I'll switch it back.